src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,19 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
     public static SessionFactory getSessionFactory() {
-        return null;
+        return instance;
+    }
+
+    private static SessionFactory initSessionFactory() {
+        return new Configuration().configure().buildSessionFactory();
```

Comment:
Ensure that the `hibernate.cfg.xml` file is present in the classpath and correctly configured to avoid runtime issues when building the `SessionFactory`.